tests/unit/rss/test_views.py

```diff
@@ -44,6 +48,72 @@ def test_rss_updates(db_request):
     assert db_request.response.content_type == "text/xml"
 
 
+def test_rss_updates_limit(db_request):
+    db_request.params = {"limit": 2}
+
+    db_request.find_service = pretend.call_recorder(
+        lambda *args, **kwargs: pretend.stub(
+            enabled=False, csp_policy=pretend.stub(), merge=lambda _: None
+        )
+    )
+
+    db_request.session = pretend.stub()
+
+    project1 = ProjectFactory.create()
+    project2 = ProjectFactory.create()
+
+    release1 = ReleaseFactory.create(project=project1)
+    release1.created = datetime.date(2011, 1, 1)
```

Comment:
Nit: Any reason to not pass this in the `create` call above?

Reply:
No reason, I just cargo-culted this from the existing tests in this file.